index.bs

```diff
@@ -4077,8 +4077,8 @@ Methods</h4>
 			4. Send a <a>control message</a> to the associated {{AudioContext}} to
 				<a href="#context-resume">run it in the rendering thread</a> only when
 				the following conditions are met:
```

Comment:
To make it explicit, can you also change this sentence to say "all the following conditions..."?